src/lib/components/chat/ChatWindow.svelte

```diff
@@ -212,7 +212,7 @@
 								placeholder="Ask anything"
 								bind:value={message}
 								on:submit={handleSubmit}
-								on:keypress={(ev) => {
```

Comment:
`keypress` was not firing on mobile. We would need more general event (i.e. `beforeinput`). It also means that on desktop: user can paste a text & would not receive LoginModal since paste event would not fire keypress. `beforeinput` would fire on all forms of input: including keypress & paste

Reply:
https://caniuse.com/?search=beforeinput

![image](https://github.com/huggingface/chat-ui/assets/11827707/11247cb0-ebf5-4b74-b56f-44e09015f82e)

Reply:
Yes, my bade, I misread the change.